SECURITY.md

```diff
@@ -1,27 +1,62 @@
+
+
+- For direct dependencies - After identifying a version of the package that is both compatible with OpenSearch Dashboards and includes a fix for the vulnerability, update the dependency in `package.json` and run `yarn osd bootstrap` to build the project and update the `yarn.lock` file.
+
+- For nested dependencies (sub-dependencies)
+
+    - If the package range is suitable but `yarn.lock` does not include the desired version, edit the `yarn.lock` file and remove the lines corresponding to the package. Then, run `yarn osd bootstrap` to have the latest version of the package, that satisfies the range from `package.json`, added to `yarn.lock`.
+
+    - If the package range is not suitable and there is no resolutions or the package range in the resolutions is not correct, we can add or update version in the resolutions for all the package sub-deps or specific package sub-dep. For more on version updates please see [Why](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-why-would-you-want-to-do-this) and [How](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-how-to-use-it) to upgrade.
+
+    ```
+    Example: If foobar@1.x exists for subdeps in yarn.lock file. But foobar@1.y is the target version.
+
+    Step 1: remove the entry from lockfile and bootstrap; if a suitable version was installed, you can stop.
+    Step 2: use resolutions to require a specific range for the nested dependency in package.json:
+    'resolutions': { "**/foobar": "^1.y",
+                     "**/foo": "^2.x" ,
+                     "**/bar": "^3.k"}
+    Step 3: repeat step 1
+
+    ```
+
+    Please be aware of that fixing nested dependencies can be tricky. Sometimes, bumping a parent package of the nested dependency can upgrade several of the nested dependencies at once to solve multiple security vulnerabilities and provide a more maintainable code base.
+
+- To backport a pull request for a CVE fix to previous versions of OpenSearch Dashboards, add the desired backport labels (ex: `backport 1.x`) to the PR. When the PR is merged, a workflow will attempt to backport the PR. If this backporting attempt fail, the PR will be updated with comments about the failure; follow the instructions in those comments to manually backport your changes which you can then use in a new PR. Here are some steps for reference.
+
+    - Identify the pull request you want to backport and the target backport version.
+    - Create a new local branch from the target version.
+    - Cherry-pick the changes from the pull request into the new branch. To do this, you can use the `git cherry-pick` command followed by the hash of the pull request commit. For example: `git cherry-pick 123456`.
+    - Resolve any conflicts. This step may require some manual intervention.
+    - Test the changes thoroughly.
```

Comment:
This is missing an explicit step to run `yarn osd bootstrap`. Because `yarn.lock` is intended to always be a generated file, this helps catch bad merges or conflict resolutions.

Reply:
will add